docs/conf.py

```diff
@@ -81,10 +81,17 @@
     "**venv",
     "base",
     "sphinx-resources",
+    # These RST files are explicitly excluded here because they are included by
+    # other files - without this exclusion, Sphinx will complain about duplicate
+    # labels.
+    "common/craft-parts/explanation/overlay_parameters.rst",
```

Comment:
Can we exclude these with `common/**`?

Reply:
We could, but I think that has more drawbacks than advantages. An advantage of having to spell out every file is that sphinx will then complain if you add a file in `common/` but forget to reference it in a TOC. This actually made me realize that a bunch of how-tos and explanations had never been linked in [rockcraft's docs](https://github.com/canonical/rockcraft/pull/515/files#diff-c92bad01f08eb61a41f190d07ad34d0829592613d9336e342083237faabdcc91R11), for example.

Reply:
Yep, agreed. Thanks for the explanation